examples/complete/variables.tf

```diff
@@ -504,3 +504,28 @@ variable "codepipeline_s3_bucket_force_destroy" {
   description = "A boolean that indicates all objects should be deleted from the CodePipeline artifact store S3 bucket so that the bucket can be destroyed without error"
   default     = false
 }
+
+variable "linux_parameters" {
```

Comment:
Please do not upgrade to 1.3. we're not ready to do this yet.

Many consumers are still using older tf versions and we've run into issues with the optional argument.

Please correct me if I'm wrong @Nuru @mcalhoun @osterman @aknysh .

This has come up several times and it would be nice if we had a decision and a link to send when it comes up.

Reply:
Ok - for now I will leave this PR pending until a decision to upgrade is made because this complex object is very annoying for a user to manage without optional properties.

Reply:
Using type any is the best way forward for now